packages/babel-plugin-import-jsx-pragma/README.md

```diff
@@ -30,6 +30,8 @@ module.exports = {
 };
 ```
 
+_Note:_ `@wordpress/babel-plugin-import-jsx-pragma` is now included in `@wordpress/babel-preset-default` (default preset for WordPress development). If you are using it, you shouldn't need to include this plugin anymore in your Babel config. 
```

Comment:
This reads as a temporary note which won't age well. Do we have a planned schedule on if / when it would no longer be valuable to include?

Reply:
I didn't think about it initially. However, I agree with your point, we should remove it in a month or something like that.

Reply:
> I didn't think about it initially. However, I agree with your point, we should remove it in a month or something like that. I added it to my todo list. Should I open PR with a related description?

I don't have a strong preference on timeline; I personally don't think the note was needed in the first place† 😅 A month can be fine. If we're soon to publish packages going into feature freeze, I suppose it could be merged immediately afterward, with the assumption the next publish wouldn't be for some time. Otherwise it becomes an issue of how do we keep the schedule.

† To me, the perfect scenario would be that the plugin gracefully handles the case that it's included twice. The next best thing would be a clear note that the preset and plugin cannot be used in combination. Re-reading this text, I think it's fairly clear. It may have just been that the phrases "now" and "anymore" caught my attention as sounding temporary and not associated with any sense of when "now" actually occurred. "As of vX.X.X" might have served as an alternative pinned to a specific point in time.

Reply:
Another try in #14530 😅